roles/confluent.common/tasks/debian.yml

```diff
@@ -107,6 +107,7 @@
     - install_java|bool
     - repository_configuration == 'confluent'
     - ansible_distribution_release == "buster"
+    - add_sid_repo|bool
```

Comment:
If this is only for certain OS / version. Does it make sense to have additional checks to make it more user friendly?

Reply:
What type of additional checks are you asking about? I've added add_sid_repo value false by default. The customer would need to provide that as true to add Sid Repo